### PR TITLE
Update Integration Tests

### PR DESCRIPTION
**Pull request by rickdonato**
To resolve integration test issues around collection versions this PR:
* Pins the Ansible version
* Upgrades the Netbox Ansible collection.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -73,8 +73,8 @@ jobs:
       before_install: >
         echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin || travis_terminate 1
       before_script:
-        - "pip install invoke toml poetry ansible pynetbox"
-        - "ansible-galaxy collection install git+https://github.com/netbox-community/ansible_modules.git,v1.2.0"
+        - "pip install invoke toml poetry pynetbox ansible==2.10.7"
```
Comment (jvanderaa) [inline .travis.yml:76]:
```suggestion
        - "pip install invoke toml poetry pynetbox ansible-base==2.10.7"
```

File: .travis.yml [hunk 2]
```diff
@@ -86,8 +86,8 @@ jobs:
...
```
Comment (jvanderaa) [inline .travis.yml:89]:
```suggestion
        - "pip install invoke toml poetry pynetbox ansible-base==2.10.7"
```

**Review by jvanderaa — COMMENTED** [review]
If we go with `ansible-base` instead of Ansible we cut out a bunch of unneeded collections, and this is what Ansible 3.x is using anyhow.

**Comment by rickdonato** [comment]
> Looks like we need to add at least one other collection/plugin though:
> 
> > fatal: [sjc-bb-01]: FAILED! => {"msg": "the connection plugin 'network_cli' was not found"}

With the the revert of the commit recommendations from @jvanderaa this comment is no longer relevant.

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**